quarkus/distribution/scripts/run.sh

```diff
@@ -16,24 +16,30 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-#
-
-# Linux Quarkus fast-jar run script for Apache Polaris
 
-set -e
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+cd "${SCRIPT_DIR}"
 
-script_dir="$(dirname "$0")"
+# Default to server if no component specified
+COMPONENT=${1:-server}
 
-if [ -z "$JAVA_HOME" ] ; then
-  JAVACMD="`\\unset -f command; \\command -v java`"
-else
-  JAVACMD="$JAVA_HOME/bin/java"
+if [ "$COMPONENT" != "server" ] && [ "$COMPONENT" != "admin" ]; then
+    echo "Usage: $0 [server|admin] [additional arguments...]"
+    exit 1
 fi
 
-if [ ! -x "$JAVACMD" ] ; then
-  echo "The JAVA_HOME environment variable is not defined correctly," >&2
-  echo "this environment variable is needed to run this program." >&2
-  exit 1
-fi
+# Shift off the first argument so $@ contains remaining args
+shift
+
+# Common JVM options
+JAVA_OPTS="${JAVA_OPTS} -Dquarkus.http.host=0.0.0.0"
```

Comment:
Why is `quarkus.http.host=0.0.0.0` necessary?

Also, being at the end, it is not overridable by users, I guess :thinking:

Reply:
Removed